app/assets/javascripts/answers.js

```diff
@@ -20,12 +20,23 @@ $(document).ready(function(e){
   $("#answers").hide();
   $("#response").hide();
 
+   if($('#answer_text').val() ==  "") 
+   $('#submit_answer').attr('disabled', true);
+
+  $('#answer_text').keyup(function(){
+    if($('#answer_text').val() !=  "") 
```

Comment:
* Again, two spaces between operator (`!=`) and operand (`""`), and a space could go after the `if`.
* Also, as noted above, an equivalent is `if ($('#answer_text').val())`.